pkg/lib/cockpit-components-table.scss

```diff
@@ -1,5 +1,4 @@
-@use "@patternfly/patternfly/base/patternfly-variables.scss";
-@import "global-variables";
+@use "global-variables" as *;
```

Comment:
So the solution here is that `@use` namespaces the import to this file right?

Reply:
No, the solution is to not import patternfly variables. (Removing the first line.)

Use is more modern and generally better than import for SASS, but it _probably_ doesn't really matter in this PR.